pkg/trace/stats/aggregation.go

```diff
@@ -99,4 +119,9 @@ func NewAggregationFromGroup(g *pb.ClientGroupedStats) Aggregation {
 			Synthetics:  g.Synthetics,
 		},
 	}
+
+	if g.ExtraTags != nil {
+		agg.ExtraTagsKey = strings.Join(g.ExtraTags, ",")
```

Comment:
These would have to be sorted.

Reply:
True, this assumes that the client sent a sorted tag string.